src/App.vue

```diff
@@ -340,6 +436,15 @@ function resetModelPosition() {
     camera.lookAt(iniCamPosition)
   }
 }
+
+function debug(output) {
+  if (debugIsEnabled) {
+    scene.add(new AxesHelper(10))
+    console.log('####### DEBUG #######')
```

Comment:
would be good to add context in the log as well.
```suggestion
    console.log('####### DEBUG 3D MODEL #######')
```